spec/presenters/hyrax/image_presenter_spec.rb

```diff
@@ -0,0 +1,8 @@
+# Generated via
+#  `rails generate hyrax:work Image`
+
+RSpec.describe Hyrax::ImagePresenter do
+  it "has tests" do
+    skip "Add your tests here"
+  end
+end
```

Comment:
Are there tests for the class of the substituted presenter?  They probably exist in another file not touched in this PR.